deployment/src/test/java/io/quarkiverse/kafkastreamsprocessor/kafka/streams/test/KafkaStreamsProcessorProcessorTest.java

```diff
@@ -70,6 +79,10 @@ private static void checkProperClassesAreRegistered() {
                 .flatMap(c -> c.getClassNames().stream())
                 .collect(Collectors.toList());
 
+        assertThat(topologyConfigBuildItem.getSinkToTopicMapping(), equalTo(Map.of("emitter-channel", "pong-events")));
```

Comment:
Move these assertions to a dedicated test method, `checkProperClassesAreRegistered` is meant to verify `@RegisterForReflection`.

Reply:
I ll create a new method to test it.
But it can't be done in a classic @Test method otherwise it will be null.